media/js/base/site.js

```diff
@@ -64,10 +64,7 @@
             if (pf.indexOf("Mac") !== -1) {
                 return 'oldmac';
             }
-            if (pf === '' &&
-                    /Firefox/.test(ua) &&
-                    /Mobile|Tablet/.test(ua) &&
-                    !(/Android/.test(ua))) {
+            if (/Firefox/.test(ua)) {
```

Comment:
I don't think we should remove `pf === ''` as this was specific to Firefox OS. Only testing for `/firefox` in the UA and relying on the ordering of these conditionals feels a little brittle?